### PR TITLE
Include file reading into Archiver

### PR DESCRIPTION
**Pull request by RedheadPoof**
Include file reading into Archiver and replace main using new function
Renamed some wrong names objects

### CODE REVIEW

File: Archiver.py [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+
+
+def read_file(file_path):
+    file_path = os.path.abspath(file_path)
+    file_name = os.path.basename(file_path).split(".")[0]
+    try:
+        with open(file_path, "r") as input_file:
+            out_string = input_file.readline()
+    except FileNotFoundError:
+        print("So file is not found, try again")
```
Comment (nicolre239) [inline Archiver.py:12]: Such file not found
Requested file not found 
File not found
etc

Actually, it's a good idea to write your comments in english first and than try to translate them via some translator to ensure Your sentences were correct

File: Archiver.py [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+        out_string = None
+        file_name = None
+    return out_string, file_name
+
+
+def write_file(final_string, input_file_name, zipping=False):
+    if zipping:
+        output_file_name = input_file_name + "_uncompressed.txt"
```
Comment (nicolre239) [inline Archiver.py:20]: Can't see anything about this "_uncompressed" postfix in requirements

File: Archiver.py [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+    else:
+        output_file_name = input_file_name + "_compressed.txt"
```
Comment (nicolre239) [inline Archiver.py:22]: Please check the  requirements and use correct postfix

File: Archiver.py [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+        with open(file_path, "r") as input_file:
+            out_string = input_file.readline()
+    except FileNotFoundError:
+        print("So file is not found, try again")
+        out_string = None
+        file_name = None
+    return out_string, file_name
+
+
+def write_file(final_string, input_file_name, zipping=False):
```
Comment (nicolre239) [inline Archiver.py:18]: Zipping = True means we are decompressing the file?
This makes not much sense, to be honest.
The opposite naming makes much more sense. Like, if 'zipping' = True  - we are compressing

**Review by nicolre239 — CHANGES_REQUESTED** [review]

File: Archiver.py (hunk 2 above)
Comment (RedheadPoof) [reply Archiver.py:20]: Ok, what about name after unzip? It should be different from the original.

File: Archiver.py (hunk 1 above)
Comment (RedheadPoof) [reply Archiver.py:12]: Agree fixed. I will  translate sentences into russian later before using them.

File: Archiver.py (hunk 3 above)
Comment (RedheadPoof) [reply Archiver.py:22]: Check, fixed.

File: Archiver.py (hunk 4 above)
Comment (RedheadPoof) [reply Archiver.py:18]: Yes, my English is bad and I'm not good in names. But opposite naming is really wrong there.

**Review by nicolre239 — APPROVED** [review]

**Pull request merged**